roles/v2plugin/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+---
+# This role contains tasks for starting docker plugin service
+
+- name: setup iptables for contiv network control plane
+  shell: >
+      ( iptables -L INPUT | grep "{{ netplugin_rule_comment }} ({{ item }})" ) || \
+      iptables -I INPUT 1 -p tcp --dport {{ item }} -j ACCEPT -m comment --comment "{{ netplugin_rule_comment }} ({{ item }})"
+  with_items:
+    - "{{ ofnet_master_port }}"
+    - "{{ ofnet_agent_port1 }}"
+    - "{{ ofnet_agent_port2 }}"
+    - "{{ netmaster_port }}"
+    - "{{ gobgp_grpc_port }}"
+    - "{{ bgp_port }}"
+
+- name: setup netmaster host alias
+  lineinfile:
+    dest: /etc/hosts
+    line: "{{ service_vip }} netmaster"
+    regexp: " netmaster$"
+    state: present
+
+- name: install v2plugin on {{ run_as }} nodes
+  shell: >
+    /usr/bin/docker plugin install --grant-all-permissions {{ contiv_v2plugin_image }} ctrl_ip={{ node_addr }} control_url={{ node_addr }}:9999 vxlan_port=8472 iflist={{ netplugin_if }} plugin_name={{contiv_v2plugin_image}} cluster_store={{cluster_store}} plugin_role={{ run_as }}
```

Comment:
Can you move these ports to defaults as variables. 9999 already has netmaster_port var.